sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/ComponentSamples.cs

```diff
@@ -76,21 +76,21 @@ public async Task RunSamplesAsync(DigitalTwinsClient client)
 
             string basicDtPayload = JsonSerializer.Serialize(basicTwin);
```

Comment:
Why is it being serialized still? I thought the idea is that they pass us an actual object, which we now serialize using the `ObjectSerializer`?

Reply:
I talked with Basel about this, and he is fixing it. Right now, this PR still has the digital twins client taking in Strings rather than T's for APIs like createDigitalTwin. Soon they will take in T's instead, and this serialization will be done in the convenience layer instead